src/hotspot/os/linux/cgroupV2Subsystem_linux.cpp

```diff
@@ -249,3 +259,15 @@ jlong CgroupV2Subsystem::pids_max() {
   return limit_from_str(pidsmax_str);
 }
 
+/* pids_current
+ *
+ * The number of tasks currently in the cgroup (and its descendants) of the process
+ *
+ * return:
+ *    current number of tasks
+ *    OSCONTAINER_ERROR for not supported
+ */
+jlong CgroupV2Subsystem::pids_current() {
```

Comment:
This should use something akin to `memory_usage_in_bytes` in this class. `pids_current_val()` isn't needed.